common/js/index-buttons.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+var keyname = 'SolidServerRootRedirectLink';
+function register() { 
+    alert(2); window.location.href = "/register"; 
+}
+document.addEventListener('DOMContentLoaded', async function() {
+    if (!UI.authn.currentUser()) await UI.authn.checkUser();
+    let user = UI.authn.currentUser();
+
+    // IF LOGGED IN: SET SolidServerRootRedirectLink. LOGOUT
+    if( user ) {
+        window.localStorage.setItem(keyname, user.uri);
+        await UI.authn.authSession.logout();
+    }
+    else {
+        let webId = window.localStorage.getItem(keyname);
+
+        // IF NOT LOGGED IN AND COOKIE EXISTS: REMOVE COOKIE, HIDE WELCOME, SHOW LINK TO PROFILE
+        if( webId ) {
+        window.localStorage.removeItem(keyname);
+        document.getElementById('loggedIn').style.display = "block";
+        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>. Visit your profile to log into your Pod.</p>`;
```

Comment:
Will make the instruction clearer.
```suggestion
        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>.</p> <p>Visit your profile to log into your Pod.</p>`;
```